migration/migrator/migrator.go

```diff
@@ -323,3 +323,18 @@ func dropTables(ctx context.Context, conn *pgx.Conn, table *schema.Table) error
 	}
 	return nil
 }
+
+var ErrNoSchema = errors.New("CURRENT_SCHEMA seems empty, possibly due to empty search_path. Try `GRANT ALL PRIVILEGES ON <search_path> TO <user>`")
+
+func convertMigrateError(err error) error {
+	if err == nil {
+		return err
+	}
+
+	// https://github.com/golang-migrate/migrate/issues/696
+	if err == migratepostgres.ErrNoSchema || strings.Contains(err.Error(), `"current_schema": converting NULL to string`) {
+		return ErrNoSchema
```

Comment:
maybe return the error with the correct search path?

Reply:
hardcode `public`?

Reply:
Done